pkg/api/validation/validation.go

```diff
@@ -397,6 +401,20 @@ func validateISCSIVolumeSource(iscsi *api.ISCSIVolumeSource) errs.ValidationErro
 	return allErrs
 }
 
+func validateFCVolumeSource(fc *api.FCVolumeSource) errs.ValidationErrorList {
+	allErrs := errs.ValidationErrorList{}
+	if len(fc.TargetWWNs) < 1 {
+		allErrs = append(allErrs, errs.NewFieldRequired("targetWWNs"))
+	}
+	if fc.FSType == "" {
+		allErrs = append(allErrs, errs.NewFieldRequired("fsType"))
+	}
+	if fc.Lun < 0 || fc.Lun > 255 {
```

Comment:
I don't see how that is working - if it is not specified, Go defaults it to 0.  You're saying 0 is a valid value.